src/app/globals.css

```
@@ -49,6 +49,13 @@
 }
 
 :root {
+  /*
+   * Next's metric-adjusted local fallbacks paint without a font request. The
+   * exact self-hosted brand faces replace them during browser idle time.
+   */
+  --active-font-sans: "Geist Fallback", ui-sans-serif, system-ui, sans-serif;
+  --active-font-mono: "Geist Mono Fallback", ui-monospace, monospace;
+  --active-font-heading: "Instrument Serif Fallback", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)